core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -284,20 +286,26 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+
     $scope.suppressDefaultAnswerGroupWarnings = function() {
       var interactionId = $scope.getCurrentInteractionId();
+      var answerGroups = responsesService.getAnswerGroups();
+      var handledAnswersArray = [];
+      var choiceIndices = [];
+      var answerChoices = [];
+      var choiceRules = [];
+      var customizationArgs = stateCustomizationArgsService.savedMemento;
```

Comment:
Thanks @prasanna08 I had a similar problem with the numChoices variable and couldn't have it out in the open with the other variables. So I'm refactoring to put maxSelectionCount inside of the check for whether the interaction is ItemSelectionInput